Powertrain builder GUI (#133)
florentLutz/FAST-OAD-CS23-HE: docs/documentation/powertrain_builder/pt_build/index.rst

.. _builder:

===============
PT file Builder
===============
The powertrain builder graphical user interface provides a convenient way to create and edit powertrain

> configurations for the FAST-GA-HE framework. Once the powertrain configuration is created, it can be saved as a PT file

Reviewer: ```suggestion
configurations for the FAST-OAD-CS23-HE framework. Once the powertrain configuration is created, it can be saved as a PT file
```

> in YAML format, which can be use in later aircraft sizing or optimization.

Reviewer: ```suggestion
in YAML format, which can be used in later aircraft sizing or optimization.
```

.. toctree::
   :maxdepth: 1

    Component node placement <node_placement>
    Component connections <component_connections>
    Delete, undo and redo <undo_redo_delete>
    Node properties configuration <node_properties>
    Save & reload design <save_load>
    End Session <end_session>

To launch the builder, add the following code snippet to your Python script:

.. code:: python

    from fastga_he.gui.power_train_builder import PowertrainBuilderLauncher

>     PowertrainBuilderLauncher.launch()

Reviewer: I wonder if this could and should be launched via Command Line Interface (CLI)

> Once loaded from existed design the builder should look like this:

Reviewer: ```suggestion
Once loaded from existing design, the builder should look like this:
```
